### PR TITLE
Service account support

### PR DESCRIPTION
**Pull request by adambird**
Added strategy for authenticating with Service Accounts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,11 +36,34 @@ end
...
+```
+
+More information in Service Accounts documentation (link to follow).
+
 ## Configuration
 
 Configurable options
 
-* `scope`: A space-separated list of permissions you want to request from the user. See the [API Authorization documentation](http://www.cronofy.com/developers/api#authorization) for a full list of available permissions.
+* `scope`: A space-separated list of permissions you want to request for the authorization. See the [API Authorization documentation](http://www.cronofy.com/developers/api#authorization) for a full list of available permissions.
```
Comment (stephenbinns) [inline README.md]: As we've not published the documentation for these scopes we can't fix this properly but they aren't going to be the standard set

Comment (adambird) [inline README.md]:
```
A space-separated list of permissions you want to request from the user. See the [API Authorization documentation](http://www.cronofy.com/developers/api#authorization) for a full list of available end-user permissions and some_other_link for Service Account permissions.
```

?

Comment (stephenbinns) [inline README.md]: The end user scopes are only relevant to the delegated scopes so probably better to only link on on that. Its a pain we haven't got a link for the service account scopes yet. 

I'd go with something like

> A space-separated list of permissions you want to request for the service account. See the Service Account documentation for the full list.

Comment (adambird) [inline README.md]: This is serving both purposes, end-user and service account. Will split

**Comment by stephenbinns** [comment]
👍

**Pull request merged**